src/createServer.js

```diff
@@ -1,11 +1,280 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
   // Use express to create a server
   // Add a routes to the server
   // Return the server (express app)
+
+  const app = express();
+
+  const users = [];
+
+  app.get('/users', express.json(), (req, res) => {
+    if (!users) {
+      res.statusCode = 200;
+
+      res.send([]);
+
+      return;
+    }
+
+    res.statusCode = 200;
+
+    res.send(users);
+  });
+
+  app.post('/users', express.json(), (req, res) => {
+    const { name } = req.body;
+
+    if (!name) {
+      res.statusCode = 400;
```

Comment:
use `sendStatus` here. Do the same all over this task code and in other projects if you would need it.